src/pudl/metadata/resources/sec10k.py

```diff
@@ -0,0 +1,100 @@
+"""Table definitions for the SEC10k tables."""
+
+from typing import Any
+
+RESOURCE_METADATA: dict[str, dict[str, Any]] = {
+    "core_sec10k__filings": {
+        "description": "Metadata describing all submitted SEC 10k filings.",
+        "schema": {
+            "fields": [
+                "filename_sec10k",
+                "central_index_key",
+                "company_name",
+                "sec10k_version",
+                "date_filed",
+                "exhibit_21_version",
+                "report_date",
+            ],
+            "primary_key": [
+                "filename_sec10k",
+            ],
+        },
+        "sources": ["sec10k"],
+        "etl_group": "pudl_models",
+        "create_database_schema": False,
+        "field_namespace": "sec10k",
+    },
+    "core_sec10k__exhibit_21_company_ownership": {
+        "description": "Company ownership data extracted from Exhibit 21 attachments to SEC 10k filings.",
+        "schema": {
+            "fields": [
+                "filename_sec10k",
+                "subsidiary_company_name",
+                "subsidiary_location",
+                "fraction_owned",
+                "report_date",
+            ],
+        },
+        "sources": ["sec10k"],
+        "etl_group": "pudl_models",
+        "create_database_schema": False,
+        "field_namespace": "sec10k",
+    },
+    "core_sec10k__company_information": {
+        "description": "Company information extracted from SEC 10k filings.",
+        "schema": {
+            "fields": [
+                "filename_sec10k",
+                "filer_count",
+                "company_information_block",
+                "company_information_block_count",
+                "company_information_fact_name",
+                "company_information_fact_value",
+                "report_date",
+            ],
+            "primary_key": [
+                "filename_sec10k",
+                "filer_count",
+                "company_information_block",
+                "company_information_block_count",
+                "company_information_fact_name",
+                "company_information_fact_value",
+            ],
+        },
+        "sources": ["sec10k"],
+        "etl_group": "pudl_models",
+        "create_database_schema": False,
+        "field_namespace": "sec10k",
+    },
+    "out_sec10k__parents_and_subsidiaries": {
+        "description": "Denormalized table containing SEC 10k company information with mapping between subsidiary and parent companies, as well as a linkage to EIA companies.",
+        "schema": {
+            "fields": [
+                "company_id_sec",
+                "filename_sec10k",
+                "report_date",
+                "report_year",
```

Comment:
Why both `report_date` and `report_year`?

Currently we've standardized on `report_date` elsewhere in PUDL regardless of reporting frequency, but have yet to go back and fix all the FERC tables that still have `report_year`

Reply:
`report_year` was there because the upstream table is partitioned by year, but no need to distribute it, so I've dropped it.